backend/config/locales/zu.yml

```diff
@@ -22,8 +22,9 @@ zu:
       verified: Verified
       unverified: Unverified
       search_placeholder: Search
-      apply: Apply filters
-      reset: Reset filters
+      apply: Apply
```

Comment:
I have slightly shorted text, because there is not much space at top of every table and I believe it is quite clear what Apply or Reset means.

Reply:
Filters will be anyway refactored to match designs later. I think in internal copy, I saw them in popover.

Reply:
Noooo :D, I have some wrong internal copy, because design from my figma file more or less match what we have now (ofc it is not pixel perfect :))

Reply:
I checked the one that is on slack channel, but that is ok we will iterate.